src/Trakx.Shrimpy.Core.Tests/Integration/ShrimpyClientTestsBase.cs

```diff
@@ -20,14 +22,6 @@ public ShrimpyClientTestsBase(ShrimpyApiFixture apiFixture, ITestOutputHelper ou
         }
     }
 
-    [CollectionDefinition(nameof(ApiTestCollection))]
```

Comment:
So these tests collections are overengineering you think?

Reply:
Because the Fixtures are in a different assembly to the actual tests xunit does not support this. I have to use IAssemblyFixure